heka/files/toml/output/influxdb.toml

```diff
@@ -2,9 +2,11 @@
 type = "HttpOutput"
 message_matcher = "Fields[payload_type] == 'txt' && Fields[payload_name] == 'influxdb'"
 encoder = "influxdb_encoder"
-address = "http://{{ output.host }}:8086/write?db=lma&precision=ms"
-username = "influxdb"
-password = "influxdbpass"
+address = "http://{{ output.host }}:{{ output.port }}/write?db={{ output.database }}&precision=ms"
```

Comment:
"...&precision=ms". How do we keep it consistent with the value configured in the InfluxDB accumulator filter?